### PR TITLE
added GitHub Actions workflow

### PR DESCRIPTION
**Pull request by voigtst**
Added the GitHub Actions workflow for the pytest testcases and the Flask mock server.
The workflow handles linting of the code with flake8 and running the testcases afterwards.
The workflow will be triggered at each push and pull request to master.
I also removed some leftover imports from the testcases.

### CODE REVIEW

File: .github/workflows/testcoverage.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: testcoverage
```
Comment (viennaa) [inline .github/workflows/testcoverage.yml:1]: this ain't a coverage test.

**Review by viennaa — CHANGES_REQUESTED** [review]
Thanks for implementing the CI action. I like the very little, though colorful output of the results.
Just a little naming mistake, then we can enable it.

File: .github/workflows/testcoverage.yml (hunk 1 above)
Comment (voigtst) [reply .github/workflows/testcoverage.yml]: This is true. Renamed it to a more appropriate name.

File: .github/workflows/integration_test.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Integration test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  VCSA-Exporter_testing:
```
Comment (viennaa) [inline .github/workflows/integration_test.yml:10]:
```suggestion
  vcsa_exporter_test:
```

**Review by viennaa — APPROVED** [review]
Good to go.

**Pull request merged**